.github/scripts/print_pr_details.py

```diff
@@ -0,0 +1,55 @@
+# .github/scripts/print_pr_details.py
+from github import Github
+import os
+
+# Initialize the GitHub client
+token = os.getenv('GITHUB_TOKEN')
+g = Github(token)
+
+# Get repository information from environment variables
```

Comment:
Missing error handling for cases where environment variables GITHUB_TOKEN or GITHUB_REPOSITORY are not set